shells/pipes-shell/source/testDevice.js

```diff
@@ -48,29 +90,28 @@ export const smokeTest = async (paths, storage, manifest, bus) => {
   //
   const notificationTest = () => {
     // spawn an arc
-    send({message: 'spawn', modality: 'dom', recipe: 'Notification'});
+    send({message: 'runArc', arcId: 'pipe-notification-test', modality: 'dom', recipe: 'Notification'});
   };
   //
   const parseTest = () => {
     // parse manifest content
-    send({message: 'parse', content: `import 'https://$particles/canonical.arcs'`});
+    send({message: 'parse', content: `import 'https://$particles/PipeApps/PipeApps.arcs'`});
```

Comment:
just a question - is the PipeApps/canonical.arcs ever used, or should it be removed?

Reply:
I have been in the habit of naming a default manifest `<folder-name>.arcs` (if I make one). Its helpful if the manifest identifies itself specifically.

`canonical.arcs` can be removed, unless the above is wrong-think.